controller/login.go

```diff
@@ -157,11 +202,19 @@ func (c *LoginController) Refresh(ctx *app.RefreshLoginContext) error {
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{
 			"err": err,
-		}, "Unable to get Keycloak token endpoint URL")
+		}, "unable to get Keycloak token endpoint URL")
 		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to get Keycloak token endpoint URL")))
 	}
 
-	rpt, err := auth.GetEntitlement(ctx, entitlementEndpoint, nil, *token.AccessToken)
+	resources, err := c.getEntitlementResourceRequestPayload(ctx, token.AccessToken)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "failed to obtain create entitlement resource request ")
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
+	}
+
+	rpt, err := auth.GetEntitlement(ctx, entitlementEndpoint, resources, *token.AccessToken)
```

Comment:
Hi @alexeykazakov , I didn't understand. if `resources == nil`, then the code falls back to the old behaviour of making the following call `auth.GetEntitlement(ctx, entitlementEndpoint, nil, *token.AccessToken)` to fetch all the resources. 

As mentioned in another comment, I will consume the changes to cap the size of RPT after the other PR is merged.